src/main/java/com/minecolonies/coremod/util/TeleportHelper.java

```diff
@@ -0,0 +1,45 @@
+package com.minecolonies.coremod.util;
+
+import com.minecolonies.coremod.entity.EntityCitizen;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+/**
+ * Helper class for teleporting citizens.
+ */
+public final class TeleportHelper
+{
+    private static final double MIDDLE_BLOCK_OFFSET = 0.5D;
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private TeleportHelper()
+    {
+        // Intentionally left empty.
+    }
+
+    public static boolean teleportCitizen(EntityCitizen citizen, World world, BlockPos location)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)